content/applications/websites/website/configuration/cookies-bar.rst

```diff
@@ -0,0 +1,55 @@
+===========
+Cookies bar
+===========
+
+Cookies are small text files sent to your device when you visit a website. They are processed and
+stored by your browser and contain information about your visit, such as login data, location,
+language, etc. There are two main types of cookies:
+
+- Essential cookies, which are necessary for the website to function properly;
+- Non-essential or optional cookies, which are used to analyze your behavior or display
+  advertisements.
+
+Notifying users about data collection, methods, and purposes is required by data protection laws
+such as `GDPR <https://gdpr.eu>`_. Cookies bars are commonly used to fulfill this obligation in a
+user-friendly and transparent manner. They are displayed immediately upon a user's first visit to
+inform visitors that the website uses cookies and let them decide whether they want to store
+non-essential cookies on their device.
+
+To enable the cookies bar on your Odoo website, go to :menuselection:`Website > Configuration` and
+enable :guilabel:`Cookies bar` in the :guilabel:`Privacy` section.
+
+Customization
+=============
+
+To adapt the display of the cookies bar, click :guilabel:`Edit` and select the :guilabel:`Cookies
+Bar` building block in the :guilabel:`Invisible Elements` at the bottom of the panel. The cookies
+bar can be customized using the options in the :guilabel:`Customize` tab in the edit panel. Three
+:guilabel:`Layouts` are available:
+
+- :guilabel:`Discrete`: thin bar
+- :guilabel:`Classic`: banner
+- :guilabel:`Popup`: you can change the popup's :guilabel:`Position` to the :guilabel:`Top`,
+  :guilabel:`Middle` or :guilabel:`Bottom` of the screen.
+
+You can also:
+
+- modify the :guilabel:`Size` of the cookies bar;
+- enable :guilabel:`Backdrop` to gray out the page in the background when the cookies bar is
+  displayed on the screen;
+- further customize the appearance of the cookies bar using :guilabel:`Block` and/or
+  :guilabel:`Column` customization options, which are available after clicking anywhere in the
+  building block.
+
+The cookies bar's contents (i.e. the consent message) can be adapted directly in the building block.
+
+Cookie policy
+=============
+
+When you enable the cookies bar for your website, Odoo creates the **Cookie Policy** page
```

Comment:
Yes, I noticed this also, and I think they just forgot to add the hyperlink to the other layouts (you can actually see something's missing because the message ends with "how we use them on" and then nothing. There are other ways to access the page, I'd like to discuss this with you as well please :)